### DISCUSSION TITLE
如何 build test 文件

### DISCUSSION THREAD

**Discussion by SeibertronSS** [InternLM/lmdeploy#2464] [Q&A]
想要 run 一下 attention_test，但是加了 -DBUILD_TEST=ON 之后会报错

**Comment by SeibertronSS** [comment]
> 想要 run 一下 attention_test，但是加了 -DBUILD_TEST=ON 之后会报错

报错信息如下 
![image](https://github.com/user-attachments/assets/6ee3bc5c-f723-4cd1-b9c7-a17e127632fa)

**Comment by lvhan028** [comment]
@irexyc could you do the favor?

**Comment by irexyc** [comment]
@SeibertronSS 

我不确定原因，但是你可以把这里的 `if` 条件去掉，直接 FetchContent_Declare

https://github.com/InternLM/lmdeploy/blob/main/CMakeLists.txt#L46-L52
